From 0e8d135ddca6329b8f1b01aad11a44bd991f87ec Mon Sep 17 00:00:00 2001
From: Vincent Knecht <vincent.knecht@mailoo.org>
Date: Sat, 27 May 2023 09:10:15 +0200
Subject: [PATCH 02/12] Adapt fstab for qcom-msm8916 devices

Signed-off-by: Vincent Knecht <vincent.knecht@mailoo.org>
---
 platform/fstab/fstab.cpp | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/platform/fstab/fstab.cpp b/platform/fstab/fstab.cpp
index a514344..518429d 100644
--- a/platform/fstab/fstab.cpp
+++ b/platform/fstab/fstab.cpp
@@ -11,6 +11,8 @@
 #define __FILE_ENCRYPT__ fileencryption=aes-256-xts:aes-256-cts
 #endif
 
+#ifndef platform_qcom_msm8916
+
 system                              /system         ext4    ro,barrier=1                          wait,first_stage_mount,logical,slotselect,avb=vbmeta_system,avb_keys=/avb
 system_ext                          /system_ext     ext4    ro,barrier=1                          wait,first_stage_mount,logical,slotselect,avb=vbmeta_system
 product                             /product        ext4    ro,barrier=1                          wait,first_stage_mount,logical,slotselect,avb=vbmeta_system
@@ -31,3 +33,15 @@ vendor_dlkm                         /vendor_dlkm    ext4    ro,noatime,errors=pa
 #else
 /devices/platform/soc/<ALL>/usb<ALL>                  auto            auto    defaults            voldmanaged=usb:auto,encryptable=userdata
 #endif
+#else
+/dev/block/platform/soc@0/7824900.mmc/by-name/system    /system            ext4    noatime,ro,errors=panic wait,first_stage_mount
+/dev/block/platform/soc@0/7824900.mmc/by-name/cache     /vendor            ext4    noatime,ro,errors=panic wait,first_stage_mount
+/dev/block/platform/soc@0/7824900.mmc/by-name/userdata  /data              ext4    noatime,nosuid,nodev,barrier=1 wait,check,latemount,quota,formattable,__FILE_ENCRYPT__
+
+// For firmwares
+// /dev/block/platform/soc@0/7824900.mmc/by-name/modem    /vendor/odm/modem    vfat    ro,shortname=lower,uid=1000,gid=1000,dmask=227,fmask=337,context=u:object_r:firmware_file:s0 wait
+/dev/block/platform/soc@0/7824900.mmc/by-name/modem     /vendor/odm/modem    vfat    defaults wait
+/dev/block/platform/soc@0/7824900.mmc/by-name/persist   /vendor/odm/persist  ext4    noatime,ro,noauto_da_alloc wait
+
+/devices/platform/soc@0/7864900.mmc/mmc_host/mmc*       auto                 auto    defaults     voldmanaged=sdcard1:auto
+#endif
-- 
2.40.1

